src/sink/interface.ts

```diff
@@ -14,44 +12,6 @@ export interface Message {
     remoteAddress?: string;
 }
 
-export interface StoredMessage extends Message {
-    id: string;
-    source: SourceReference;
-    expires?: Date;
-}
-
-export interface Query {
-    namespace: string;
-    id?: string;
-    podName?: string;
-    labelSelector?: {[k: string]: string};
-}
-
-export interface RetrieveOptions {
-    limit?: number;
-    offset?: number;
-}
-
-export interface RetrieveStreamOptions {
-    onlyNew?: boolean;
-}
-
-export interface RetrieveResult {
-    messages: StoredMessage[];
-    totalCount: number;
-}
-
 export interface Parser {
```

Comment:
I couldn't find another reference to this interface. Can this be removed as well?